tests/dummy/app/pods/demo/template.hbs

```diff
@@ -1,4 +1,4 @@
-<!-- BEGIN-SNIPPET template -->
+{{! BEGIN-SNIPPET template }}
```

Comment:
Does the add-on looking for these comments work with this format (as well)?

Reply:
Yes, I had the same thought and verified after I made the change.